downstream/modules/platform/ref-platform-non-inst-database-inventory.adoc

```diff
@@ -38,7 +39,7 @@ registry_password='<registry password>'
 # Automation Hub Configuration
 #
 automationhub_admin_password='<password>'
-automationhub_pg_host='database-01.acme.org'
+automationhub_pg_host='controller'
 automationhub_pg_port='5432'
 automationhub_pg_database='automationhub'
 automationhub_pg_username='automationhub'
```

Comment:
Do you think you could put the example in a `ini/database.ini` file then reference it with an includes directive? 

Similar to what Kevin is doing here: https://github.com/RedHatInsights/red-hat-ansible-automation-platform-documentation/pull/415/files#diff-64084273a8eab3e0c8b734bd9131033536167c8a0ad2cae15ff72993ee6a08dd